pkgs/top-level/all-packages.nix

```diff
@@ -6078,7 +6078,7 @@ with pkgs;
 
   binutilsCross = assert targetPlatform != buildPlatform; lowPrio (
     if targetPlatform.libc == "libSystem" then darwin.cctools_cross
-    else forcedNativePackages.binutils.override {
+    else forcedNativePackages.binutils-raw.override {
```

Comment:
Why? `binutils-raw` is very rarely what you want, as the GNU tools in it often screw up Mach-O binaries.